DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateApprovals/Index.cshtml

```diff
@@ -4,25 +4,31 @@
 @using Microsoft.Extensions.Configuration
 @model DelegateApprovalsViewModel
 
+<link rel="stylesheet" href="@Url.Content("~/css/shared/cardWithButtons.css")" asp-append-version="true">
+
 @{
   ViewData["Title"] = "Approve delegate registrations";
 }
 
 <div class="nhsuk-grid-row">
   <div class="nhsuk-grid-column-full">
     <h1 class="nhsuk-heading-xl">Approve delegate registrations</h1>
-    @if (Model.Delegates.Any()) {
-      @foreach (var delegateApproval in Model.Delegates) {
-        <partial name="_unapprovedDelegateExpandable" model="delegateApproval"/>
+    @if (Model.Delegates.Any())
+    {
+      @foreach (var delegateApproval in Model.Delegates)
+      {
+        <partial name="_unapprovedDelegateExpandable" model="delegateApproval" />
       }
-      <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-row nhsuk-u-margin-top-7">
         <div class="nhsuk-grid-column-full">
           <a class="nhsuk-button" href="" role="button">
             Approve All

```

Comment:
Can we change this to "Approve all" to match button text capitalisation elsewhere?